lib/capybara/driver/node.rb

```diff
@@ -138,6 +156,27 @@ def inspect
       def ==(other)
         eql?(other) || (other.respond_to?(:native) && native == other.native)
       end
+
+      protected
+
+      def normalize_spacing(text)
```

Comment:
Could we include some comments here or around the constants about why we are using the constants? Thinking about future readers of this, and the temptation to reverse it because they are only used in one place 😅

Reply:
Will move all of these to a new file and comment them all fairly heavily on what/why they're around.